projects/objects/geometries/protos/Rectangle.proto

```diff
@@ -0,0 +1,57 @@
+#VRML_SIM R2019b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
```

Comment:
I believe we should release that under the Apache 2.0 license.

Reply:
This is a more general question, for now, all of our 'home-made' PROTO are using this custom license.
If we want to change this we should probably define (and document) a clear rule and switch them all at the same time.

Reply:
OK, let's do it separately.